src/Manager.js

```diff
@@ -488,43 +494,45 @@ class GiveawaysManager extends EventEmitter {
  * @param {Discord.MessageReaction} reaction The reaction to enter the giveaway
  *
  * @example
- * // This can be used to add features like removing the user reaction
+ * // This can be used to add features like removing reactions of members when they do not have a specific role (aka giveaway requirements). Best used with the `exemptMembers` property of the giveaways. 
```

Comment:
```suggestion
 * // This can be used to add features like removing reactions of members when they do not have a specific role (such as giveaway requirements). Best used with the `exemptMembers` property of the giveaways. 
```